response_operations_ui/views/passwords.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+from flask import abort, redirect, render_template, request, url_for, Blueprint, current_app as app
+from itsdangerous import URLSafeSerializer, BadSignature, SignatureExpired, BadData
+from structlog import wrap_logger
+
+from response_operations_ui.forms import ForgotPasswordForm, ResetPasswordForm
+from response_operations_ui.controllers import uaa_controller
+from response_operations_ui.controllers.notify_controller import NotifyController
+from response_operations_ui.exceptions.exceptions import NotifyError
+from response_operations_ui.common import token_decoder
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+BAD_AUTH_ERROR = 'Unauthorized user credentials'
+
+passwords_bp = Blueprint('passwords_bp', __name__, static_folder='static', template_folder='templates')
+
+
+@passwords_bp.route('/forgot-password', methods=['GET'])
+def get_forgot_password():
+    form = ForgotPasswordForm(request.form)
+    return render_template('forgot-password.html', form=form)
+
+
+@passwords_bp.route('/forgot-password', methods=['POST'])
+def post_forgot_password():
+    form = ForgotPasswordForm(request.form)
+    form.email_address.data = form.email_address.data.strip()
+    email = form.data.get('email_address')
+
+    if form.validate():
+        return send_password_change_email(email)
+
+    return render_template('forgot-password.html', form=form, email=email)
+
+
+@passwords_bp.route('/forgot-password/check-email', methods=['GET'])
+def forgot_password_check_email():
+    encoded_email = request.args.get('email', None)
```

Comment:
Small thing.  Remove the `None` on this line as `.get(thing, default)` defaults to None if it can't find it